Makefile

```diff
@@ -37,27 +45,54 @@ qa:
 	$(eval env=qa)
 	$(eval env_config=qa)
 	$(eval space=bat-qa)
+	$(eval AZ_SUBSCRIPTION=s121-findpostgraduateteachertraining-development)
 
 staging:
 	$(eval env=staging)
 	$(eval env_config=staging)
 	$(eval space=bat-staging)
-
+	$(eval AZ_SUBSCRIPTION=s121-findpostgraduateteachertraining-test)
 
 production:
 	$(if $(CONFIRM_PRODUCTION), , $(error Can only run with CONFIRM_PRODUCTION))
 	$(eval env=production)
 	$(eval env_config=production)
 	$(eval space=bat-prod)
+	$(eval AZ_SUBSCRIPTION=s121-findpostgraduateteachertraining-production)
 
 sandbox:
 	$(eval env=sandbox)
 	$(eval env_config=sandbox)
-	$(eval space=bat-sandbox)
+	$(eval space=sandbox)
+	$(eval AZ_SUBSCRIPTION=s121-findpostgraduateteachertraining-production)
+
+install-fetch-config:
+	[ ! -f bin/fetch_config.rb ] \
+		&& curl -s https://raw.githubusercontent.com/DFE-Digital/bat-platform-building-blocks/master/scripts/fetch_config/fetch_config.rb -o bin/fetch_config.rb \
+		&& chmod +x bin/fetch_config.rb \
+		|| true
+
+set-azure-account:
+	az account set -s ${AZ_SUBSCRIPTION}
 
+edit-app-secrets: install-fetch-config set-azure-account
+	. terraform/workspace-variables/$(env).sh && bin/fetch_config.rb -s azure-key-vault-secret:$${TF_VAR_key_vault_name}/$${TF_VAR_key_vault_app_secret_name} \
+		-e -d azure-key-vault-secret:$${TF_VAR_key_vault_name}/$${TF_VAR_key_vault_app_secret_name} -f yaml
```

Comment:
Please add `-c` to ask for confirmation, as in https://github.com/DFE-Digital/bat-platform-building-blocks/blob/master/scripts/fetch_config/fetch_config.rb#L40
Same for edit-infra-secrets